src/figuresClasses.ts

```diff
@@ -1,11 +1,72 @@
-export interface Figure {}
+export interface Figure {
+  shape: 'triangle' | 'circle' | 'rectangle';
+  color: 'red' | 'green' | 'blue';
+  getArea(): number;
+}
+
+export class Triangle implements Figure {
+  shape: 'triangle' = 'triangle';
+
+  constructor(
+    public color: 'red' | 'green' | 'blue',
+    private a: number,
+    private b: number,
+    private c: number,
+  ) {
+    if (a <= 0 || b <= 0 || c <= 0) {
+      throw new Error('Sides figure must be greater than 0');
+    }
+
+    if (a + b <= c || a + c <= b || b + c <= a) {
+      throw new Error('These sides cannot form a triangle.');
```

Comment:
[CODE STYLE] - The error message should be more descriptive to explain the triangle inequality theorem. For example, 'The provided side lengths do not satisfy the triangle inequality theorem and cannot form a triangle.'